package.json

```diff
@@ -11,7 +11,7 @@
     "build:dev": "webpack --mode development",
     "build:prod": "webpack --mode production",
     "build": "npm run build:dev && npm run build:prod",
-    "dev": "webpack-dev-server --mode development"
+    "dev": "webpack-dev-server --mode development --host=0.0.0.0"
```

Comment:
why the host flag?

Reply:
It's for mobile local testing:

https://stackoverflow.com/questions/35412137/how-to-get-access-to-webpack-dev-server-from-devices-in-local-network

I was able to test/dev with the dev tools mobile emulator in Firefox without any issues, but Chrome and Safari were a different story. I couldn't get touch events working on those browsers ([one](https://stackoverflow.com/questions/53804541/activate-touch-events-on-chrome-71), [two](https://github.com/fabricjs/fabric.js/issues/5362)). We can however take advantage of this method to test local environment directly on your device: 

* with the `--host` flag there, just `npm run dev` 
*  then make sure both device and laptop are connected to the same wifi
* on device, navigate to the correct IP ([android](https://stackoverflow.com/questions/4779963/how-can-i-access-my-localhost-from-my-android-device), [iphone](https://stackoverflow.com/questions/3132105/how-do-you-access-a-website-running-on-localhost-from-iphone-browser)) and then we can easily test

This is how I was able to test on Chrome and Safari. I'm still taking care of the CBT account, so not sure how it would perform there.

Reply:
I did it by attaching the `touchstart` event to the root back then, not sure why, but it works in Chrome

https://github.com/wikimedia/wikipedia-preview/blob/T240006-mobile/src/index.js#L29